server/middlewares/index.js

```diff
@@ -0,0 +1,65 @@
+const Document = require('../models/documents');
+
+module.exports = {
+  hasAccess: ((accessLevel, userId, params) => {
+    return ((req, res, next) => {
+      if (accessLevel.indexOf(req.decoded.role) > -1) {
+        if (req.decoded.role === 'user') {
+          params = req.params.user_id;
```

Comment:
You could check these directly. No need to assign a variable to them, unless you need to manipulate the data and store it for come particular reason.